### ISSUE TITLE
Wyoming dashboard showing cities not in wyoming

### ISSUE THREAD

**Issue by saikat** [Bernie-2016/VolunteerDashboard#29]
http://organize.berniesanders.com/volunteer-dashboard/#state=WY&startdate=2015-06-28&enddate=2015-12-05&view=chart

@coventry my guess is this is because we are matching to nearest CL region?  Also, any idea why "Wyoming" shows up as a region here?

**Comment by coventry** [comment]
"Wyoming" is CL region:  http://wyoming.craigslist.org/

**Comment by coventry** [comment]
One way to fix this would be to break the CL regions up by state boundaries and restrict assignment of previously unassigned zips to regions which are in the same state.

I'll take a look at how many regions this would result in.

**Comment by michelledeatrick** [comment]
Has this issue been resolved?

**Comment by coventry** [comment]
Hi, Michelle.  There are a number of ways to fix this, so if you tell me the problems it's causing it will help me to identify the proper solution.

**Comment by michelledeatrick** [comment]
Thanks, Alex. Here are the problems with the way the data currently attributed to Wyoming is being handled:
1) Of the six regions listed for Wyoming--Billings, Fort Collins, Provo, Eastidaho, logan and wyoming,--four are not in Wyoming at all (Fort Collins is in Colorado; Provo is in Utah; Eastidaho is, well, East Idaho; and Provo is in Utah). 

2) I don't have insight into the underlying data, but I believe that it's not just a cosmetic problem, that the underlying data needs to be attributed to the correct states/regions, by switching which zips are being attributed to which states/regions. 

3) And in addition, that the state of Wyoming needs to be divided into some reasonable regions/units of analysis. 

I do understand the root cause--the use of CL List for defining areas. We 

Also, having "Wyoming" as a region of "Wyoming" doesn't make sense, here:  that the local and regional organizers do not organize across state lines. So when Wyoming group organizers  go to the data for their state and see "Eastern Idaho" and "Provo,"

**Comment by coventry** [comment]
It seems reasonable to break Wyoming up by county.  Here is what that looks like: ![Wyoming pop density](https://upload.wikimedia.org/wikipedia/commons/a/a7/Wyoming_population_map.png)

Some Wyoming counties could reasonably be aggregated.  Let me know what you'd like.

**Comment by michelledeatrick** [comment]
I agree that some of these could be aggregated. I will have someone look at this asap and get back to you.

**Comment by michelledeatrick** [comment]
These regions, which are fairly standard for Wyoming, make a lot of sense. What do you think, Alex?

![image](https://cloud.githubusercontent.com/assets/16112379/11603596/f474eb82-9ab0-11e5-8042-6f18cd03f3bc.png)

**Comment by coventry** [comment]
Looks good.  I can set this up.

**Comment by coventry** [comment]
This is fixed in Bernie-2016/EventCounter@ab1c781 and coventry/cl-zip-codes@03d756b . This version of the regions is currently staged at http://gobernie.net:4999/volunteer-dashboard/
